src/utilities/filetypes/EpwFile.cpp

```diff
@@ -753,6 +792,38 @@ boost::optional<EpwDesignCondition> EpwDesignCondition::fromDesignConditionsStri
   return boost::optional<EpwDesignCondition>(dc);
 }
 
+boost::optional<EpwGroundTemperatureDepth> EpwGroundTemperatureDepth::fromGroundTemperatureDepthsString(const std::string& line) {
+  std::vector<std::string> list = splitString(line, ',');
+  return fromGroundTemperatureDepthsStrings(list);
+}
+
+boost::optional<EpwGroundTemperatureDepth> EpwGroundTemperatureDepth::fromGroundTemperatureDepthsStrings(const std::vector<std::string>& list) {
+  EpwGroundTemperatureDepth gtd;
+  // Expect 48 items in the list
+  if (list.size() < 48) {
+    LOG_FREE(Error, "openstudio.EpwFile", "Expected 48 fields in EPW ground temperature depth instead of the " << list.size() << " received");
+    return boost::none;
+  }
```

Comment:
```suggestion
  // Expect 48 items in the list
  if (list.size() != 16) {
    LOG_FREE(Error, "openstudio.EpwFile", "Expected 16 fields in EPW ground temperature depth instead of the " << list.size() << " received");
    return boost::none;
  }
```